File: Programming Basics/JavaScript/Conditional Statements Advanced/Labs/personal-titles.js
function solve(input) {
    const age = Number(input[0]);
    const gender = input[1];

    let title = "";
    if (gender === "m") {
        if (age < 16) title = "Master";
        else title = "Mr.";
    } else if (gender === "f") {
        if (age < 16) title = "Miss";
        else title = "Ms.";
    }

    console.log(title);
}

File: Programming Basics/C++/Conditional Statements Advanced/Labs/PersonalTitles/PersonalTitles.cpp
#include <iostream>
using namespace std;

int main()
{
    double age;
    string gender;
    cin >> age >> gender;

    string title = "";
    if (gender == "m")
    {
        if (age < 16) title = "Master";
        else title = "Mr.";
    }
    else if (gender == "f")
    {
        if (age < 16) title = "Miss";
        else title = "Ms.";
    }

    cout << title << endl;
}
